packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -15,7 +15,7 @@ import { childrenExist, commonPropTypes, ChildrenComponentProps } from '../../ut
 import { ComponentEventHandler } from '../../types';
 import { useAnimationStyles } from './useAnimationStyles';
 
-export type AnimationChildrenProp = (props: { classes: string }) => React.ReactNode;
+export type AnimationChildrenProp = (props: { classes: string; state: string }) => React.ReactNode;
```

Comment:
```suggestion
export type AnimationChildrenProp = (props: { classes: string; state: 'unmounted' | 'exited' | 'entering' | 'entered' | 'exiting' }) => React.ReactNode;
```

Let's do better typings there